src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,34 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    const deleteCustomer = () => {
+      const customers = customersGetter();
+      const index = customers.findIndex((customer) => customer.name === customerId);
+
+
+      if (index !== -1) {
+        const updatedCustomers = [...customers];
+        updatedCustomers.splice(index, 1);
+        customersSetter(updatedCustomers);
+      }
+    };
+
+    try {
+      deleteCustomer();
+      queryClient.invalidateQueries('customers');
+    } catch (error) {
+      console.error('An error occurred while deleting the customer:', error);
+
+    }
+  }
+  function generateActionItems(name, handleEdit, handleDelete) {
+    return [
+      { label: 'Edit', onClick: (name) => handleEdit(name) },
+      { label: 'Delete', onClick: (name) => handleDelete(name) },
```

Comment:
if you notice the documentation shows:

```
    {
      title: 'Some action',
      onClick: () => console.log(`clicked on Some action, on row ${repo.name}`)
    },
```

they are using title and not label, thats why the text isn't showing up